src/component/expense-form/expense-form.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import autoBind from '../../utils/utils';
+
+const defaultState = {
+  name: '',
+  price: '',
```

Comment:
Nice job adding in the `price` to your default state to get rid of the uncontrolled component error.